Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.9.6-slim-buster as base 
+
+RUN apt-get update \
+    && apt-get -y install curl \
+    && curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+
+COPY . /app 
+WORKDIR /app 
+ENV PATH="${PATH}:/root/.poetry/bin"
+RUN poetry install 
+EXPOSE 5000
+
+FROM base as production
+CMD ["poetry", "run", "gunicorn", "-b", "0.0.0.0:5000", "todo_app.app:create_app()"]
```

Comment:
There are other adjustments you could make for the production stage, e.g. `poetry install --no-dev` for production vs `poetry install` for development instead of the shared one in the base stage.

Or simpler example: add `ENV FLASK_ENV=production` to the production stage.